projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java

```diff
@@ -29,40 +27,39 @@
 @Path(CoordConsts.SVC_CFG_WORK_MGR2)
 @Produces(MediaType.APPLICATION_JSON)
 public class WorkMgrServiceV2 {
+
+  private BatfishLogger _logger = Main.getLogger();
+
+  @DefaultValue(CoordConsts.DEFAULT_API_KEY)
+  @HeaderParam(CoordConsts.SVC_KEY_API_KEY)
+  private String _apiKey;
+
   /** Information on the URI of a request, injected by the server framework at runtime. */
   @Context private UriInfo _uriInfo;
 
   /** Returns the list of {@link Container containers} that the given API key may access. */
   @GET
-  @Path("/containers")
-  public Response getContainers(
-      @DefaultValue(DEFAULT_API_KEY) @HeaderParam(SVC_KEY_API_KEY) String apiKey) {
-    List<Container> containers = Main.getWorkMgr().getContainers(apiKey);
+  @Path(CoordConsts.SVC_KEY_CONTAINERS)
+  public Response getContainers() {
+    _logger.info("WMS:getContainers\n");
+    List<Container> containers = Main.getWorkMgr().getContainers(_apiKey);
     return Response.ok(containers).build();
   }
 
   /** Redirect to /containers if the user does not supply a container ID. */
   @GET
-  @Path("/container")
-  public Response redirectContainer(
-      @DefaultValue(DEFAULT_API_KEY) @HeaderParam(SVC_KEY_API_KEY) String apiKey) {
+  @Path(CoordConsts.SVC_KEY_CONTAINER_NAME)
+  public Response redirectContainer() {
+    _logger.info("WMS:redirect container\n");
     return Response.status(Status.MOVED_PERMANENTLY)
-        .location(_uriInfo.getRequestUri().resolve("containers"))
+        .location(_uriInfo.getRequestUri().resolve(CoordConsts.SVC_KEY_CONTAINERS))
         .build();
   }
 
-  /** Returns information about the given {@link Container}, provided this user can access it. */
-  @GET
+  /** Relocate the request to ContainerResource. */
   @Path("/container/{id}")
```

Comment:
The separation between `WorkMgrServiceV2` and `ContainerResource` seems a little weird to me. I would think a `Container`-specific resource would handle the `/{id}` link for example.

I'm not sure of the best approach here, what do you think?